.github/workflows/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Feature Request
+description: Submit a request for new Scenic features
+labels:
+  - "type: feature"
+  - "status: triage"
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Thank you for contributing! Before submitting a feature request, please make sure it has no duplicate and/or has already been addressed by searching through [the existing and past issues](https://github.com/BerkeleyLearnVerify/Scenic/issues)
+
+  - type: textarea
+    attributes:
+      label: Describe the feature and motivation
+      description: |
+        Please provide a clear and concise proposal of the feature and outline the motivation.
+      validations:
+        required: true
+
+  - type: textarea
+    attributes:
+      label: Additional context
+      description: |
+        Add any other context, such as pseudo code, links, diagrams, screenshots, to help the community better understand the feature request.
+
+  - type: checkboxes
+    attributes:
+      label: Issue Submission Checklist
+      options:
+        - label: I checked the open issues, forum, etc. and have not found any solution
+        - label: There is reproducer code and related data files (videos, images, online drive links, etc)
```

Comment:
This item doesn't make sense for feature requests -- you can just drop it.

Reply:
Ack!